gaffer-as-a-service/README.md

```diff
@@ -25,13 +25,13 @@ To access the Eureka Dashboard when running locally visit `http://localhost:8761
   * Eureka Server
   * Your chosen side car
 * In GaaS-UI set the ENV variables in the `.env` file. For example, when running everything using JWT authentication, the env variables are set as below:
-* `REACT_APP_API_PLATFORM="OTHER"
+* ```REACT_APP_API_PLATFORM="OTHER"
```

Comment:
This still looks strange because of the bullet point

Reply:
Removed bulletpoint